web/src/Repositories/DoctrineOrmSharesRepository.php

```diff
@@ -103,13 +103,12 @@ public function remove(Share $share)
     /**
      * Saves all calendar shares. Any other existing shares will get removed
      *
-     * @param AgenDAV\CalDAV\Resource\Calendar $calendarj
+     * @param \AgenDAV\CalDAV\Resource\Calendar $calendar
      */
     public function saveFromCalendar(Calendar $calendar)
     {
-        $url = $calendar->getUrl();
         $current_shares = $this->getSharesOnCalendar($calendar);
-        foreach ($shares as $share) {
+        foreach ($current_shares as $share) {
```

Comment:
I'm not 100% sure if this is correct, please check thoroughly, but it is not correct in its original incarnation, too.